README.rst

```diff
@@ -13,6 +13,9 @@ django-image-cropping
     :target: https://coveralls.io/r/jonasundderwolf/django-image-cropping
```

Comment:
This badge is obsolete in case we drop coveralls.

Reply:
done